tmlibrary/tmlib/tools/classification.py

```diff
@@ -88,12 +136,19 @@ def process_request(self, submission_id, payload):
             label_map[float(i)] = {'name': cls['name'], 'color': cls['color']}
 
         unique_labels = np.unique(labels.values())
+
+        # Build a name for the result, max. 30 characters in total. Cuts off
+        # input names after character 20. (limited by database settings)
+        result_name = payload['name'][:20] + '-' + str(submission_id)
+
+        # Train the classifier
         result_id = self.register_result(
-            submission_id, mapobject_type_name,
-            result_type='SupervisedClassifierToolResult',
-            unique_labels=unique_labels, label_map=label_map
+           submission_id, mapobject_type_name,
+           result_type='SupervisedClassifierToolResult', name=result_name,
+           unique_labels=unique_labels, label_map=label_map
         )
-
+        logger.info('Result id')
+        logger.info(result_id)
```

Comment:
Is this still needed? In case you want to keep it, explain better what kind of "result" are you talking about (think about your sysadmins scrolling through the logs :smile:)

Is this a really important line that deserves a "info" status in the logs? Log lines are divided into many "levels", usually DEBUG, INFO, WARNING, ERROR. I'd set it as a debug line.

If you keep it, make it a single line, otherwise there may be other lines that end up in the middle, sth like:
```suggestion
        logger.debug('Result id from the classification: ' + str(result_id))
```

Reply:
Ah, one of my many logging entries made it through. No, that was debugging online. Thanks for catching this :)